src/game/Accounts/AccountMgr.cpp

```diff
@@ -37,6 +38,9 @@ AccountMgr::~AccountMgr()
 
 AccountOpResult AccountMgr::CreateAccount(std::string username, std::string password) const
 {
+    SRP6* srp = new SRP6();
+    std::string sha_pass_hash;
```

Comment:
You don't need to declare all variables at the top of a function. It's not good old C, so please move this to where it's used for the first time.

Reply:
old habbits. moved the declarations. thx